conf/radiusd/cli.conf.example

```diff
@@ -0,0 +1,16 @@
+pidfile = %%pid_file%%
+
+$INCLUDE radiusd.conf
+
+listen {
+    ipaddr = %%management_ip%%
+    port = 1814
+    type = auth
+    virtual_server = packetfence-cli
+}
+
+listen {
+    type = control
+    socket = %%socket_file%%
+    mode = rw
+}
```

Comment:
You will have to add the logging section here since we now configure it per process.